hw02-DIYarrayList/src/main/java/ru.otus/DIYarrayList.java

```diff
@@ -0,0 +1,207 @@
+package ru.otus;
+
+import java.util.*;
+
+public class DIYarrayList<T> implements List<T> {
+    private Object[] DIYarray;
```

Comment:
```suggestion
    private Object[] diyArray;
```

Поля принято писать с маленькой буквы в camelCase